src/wp-includes/block-template-utils.php

```diff
@@ -1014,6 +1027,19 @@ function _build_block_template_result_from_post( $post ) {
 		}
 	}
 
+	if ( 'wp_template' === $post->template_type ) {
```

Comment:
Did you mean
```suggestion
	if ( 'wp_template' === $post->post_type ) {
```
?
`WP_Post->template_type` is a dynamic property and is not declared on the `WP_Post` class.